overlord/managers_test.go

```diff
@@ -201,7 +201,7 @@ apps:
 	chg.AddAll(ts)
 
 	st.Unlock()
-	err = ms.o.Settle()
+	err = ms.o.Settle(5 * time.Second)
```

Comment:
If 5 * time.Second has been determined to be "good value" for the tests, why not make it the default in Settle, or a constant that every test can reference (and it's less painful to update it if we need to).

Reply:
I went back and forth a bit on that,  what is also easy is to have local settle helper with the value, that what we end up in other tests that had one before anyway. If we have to change it it would be suite or per test. Happy to revisit this later.

Reply:
heh, making it a default would make the diff much smaller :-D default args are slightly hacky in go tho

Reply:
Right. But a constant `settleTestTimeout = 5*Second` defined in a single place would do (wouldn't reduce the diff, but reduce maintenance).

Reply:
I know, not great unless really necessary, the other way is to take a *struct

Reply:
FWIW my +1 stands without any of this :-)

Reply:
I already approved before, but thanks for the extra change with settleTimeout!